ospd/main.py

```diff
@@ -159,8 +159,7 @@ def main(
     if not daemon.check():
         return 1
 
-    daemon.init()
-
+    daemon.init(server)
     daemon.run(server)
```

Comment:
```suggestion
    daemon.run()
```

Reply:
Could you address by other hint and remove the server argument from the run method of OSPDaemon?